stackslib/src/net/stackerdb/mod.rs

```diff
@@ -142,7 +142,13 @@ use crate::util_lib::boot::boot_code_id;
 use crate::util_lib::db::{DBConn, DBTx, Error as db_error};
 
 /// maximum chunk inventory size
-pub const STACKERDB_INV_MAX: u32 = 4096;
+pub const STACKERDB_INV_MAX: u32 = STACKERDB_PAGE_MAX * 2;
```

Comment:
Why is this `* 2`?

Reply:
I don't want to engage with what the theoretical limits of an inventory should be here -- just want it to be big enough to fit the needs of the `.signers` contract, which I realize this constant isn't.

It should be

```
2 * 4000 * SIGNER_SLOTS_PER_USER
```